### PR TITLE
Update SQL language configuration to handle auto-closing characters

### PR DESCRIPTION
**Pull request by kburtram**
This is a fix for this issue https://github.com/Microsoft/vscode-mssql/issues/277.

### CODE REVIEW

**Comment by MitchellSternke** [comment]
Does this mean that https://github.com/Microsoft/vscode-mssql/blob/dev/syntaxes/sql.configuration.json is no longer needed if we are configuring it using code?

File: src/languageservice/serviceclient.ts [hunk 1]
```diff
@@ -146,9 +146,43 @@ export default class SqlToolsServiceClient {
...
+     * @memberOf SqlToolsServiceClient
+     */
+    private initializeLanguageConfiguration(): void {
+        languages.setLanguageConfiguration('sql', {
+            comments: {
+                lineComment: '--',
+                blockComment: ['/*', '*/']
+            },
+
+            brackets: [
```
Comment (llali) [inline src/languageservice/serviceclient.ts]: I don't see any change in sql.configuration.json. shouldn't we remove the configurations for brackets there?

**Comment by kburtram** [comment]
I left the sql.configuration.json file there since it's referenced in the package.json to define the extension contribution points in the vscode-extension manager.  We could probably remove the contents of the file, but and have it contain an empty JSON object, but I didn't test that.

**Comment by llali** [comment]
:shipit:

**Pull request merged**